CONTRIBUTING.md

```diff
@@ -78,6 +78,12 @@ npm run dev
 You should now have an Express server running at
 [localhost:5000](http://localhost:5000) :tada:
 
+### Installation Troubleshooting
+
+If there was an error building the `node-sass` dependency, try running `npm rebuild node-sass`.
+
+**NOTE**: If errors are still shown after you have installed all the dependencies, try to delete the node modules directory in your project and run `npm install` again. 
```

Comment:
"Note" should be bolded or capitalized, not both.

`node_modules` should be spelled and formatted correctly.

```suggestion
**Note**: If errors are still shown after you have installed all the dependencies, try to delete the `node_modules` directory in your project and run `npm install` again. 
```